vllm/transformers_utils/configs/deepseek_vl.py

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) 2023-2024 DeepSeek.
```

Comment:
Could you explain in more detail why we need to patch `collections` module as well as the config?

Reply:
This model introduces the attrdict module, which will report an error（from collections import Mapping --》ImportError: cannot import name 'Mapping' from 'collections'） in Python 3.10 and above. I will try to remove this dependency.